x/bank/legacy/v0_40/types.go

```diff
@@ -1,4 +1,4 @@
-package v039
+package v040
```

Comment:
supply is missing

Reply:
yeah, noticed that too. In #6839 i'll comb through all genesis states and see if a migration is needed for each of them, if supply is not here then i'll do it there

Reply:
This PR isn't for 0.40, it's for 0.39. There is a separate issue to wrap up 0.40 which @amaurymartiny already pointed to.